old-fashioned-image-build

```diff
@@ -199,11 +204,20 @@ export MIRROR=$(egrep "(archive|ports)" /etc/apt/sources.list|head -1 | \
 export MIRROR_PATH=$(egrep "(archive|ports)" /etc/apt/sources.list|head -1 | \
                cut -d' ' -f2 | cut -d'/' -f4)
 
-/usr/share/launchpad-buildd/bin/in-target override-sources-list \
-  --backend=lxd --series=$SERIES --arch=${ARCH} $LIVEFS_NAME \
-    "deb http://$MIRROR/${MIRROR_PATH} $SERIES main restricted universe multiverse" \
-    "deb http://$MIRROR/${MIRROR_PATH} $SERIES-security main restricted universe multiverse" \
-    "deb http://$MIRROR/${MIRROR_PATH} $SERIES-updates main restricted universe multiverse"
+if [ -n "$PROPOSED" ]; then
+  /usr/share/launchpad-buildd/bin/in-target override-sources-list \
+    --backend=lxd --series=$SERIES --arch=${ARCH} $LIVEFS_NAME \
+      "deb http://$MIRROR/${MIRROR_PATH} $SERIES main restricted universe multiverse" \
+      "deb http://$MIRROR/${MIRROR_PATH} $SERIES-proposed main restricted universe multiverse" \
```

Comment:
@codyshepherd why is this change necessary?

Reply:
To enable, e.g. `bionic-proposed` based on the `$PROPOSED` flag. So that we can use old-fashioned/bartender to build against packages in the -proposed pocket.

Reply:
Otherwise packages from the -proposed pocket are not pulled in

Reply:
This is mixing two different contexts for enabling -proposed and they need to be separate.

This first change is adding the -proposed pocket in the build host container, which would pull in livecd-rootfs and other packages used to build images from -proposed.

The second change to add --proposed to `in-target buildlivefs` would enable --proposed in the images we build.

We might want to do either of these, but we need to control them separately.